apple-app-site-association.json

```diff
@@ -17,44 +17,44 @@
       {
         "appID": "23KMWZ572J.net.artsy.artsy",
         "paths": [
-          "NOT /series/*",
-          "NOT /video/*",
-          "NOT /news",
-          "NOT /news/*",
-          "NOT /editorial/*",
           "NOT /2016-year-in-art",
-          "NOT /venice-biennale*",
-          "NOT /gender-equality*",
+          "NOT /change-password",
+          "NOT /click/*/aHR0cHM6Ly9hcHAuYWRqdXN0LmNv*",
+          "NOT /click/*/aHR0cHM6Ly9hcnRzeS50eXBlZm9ybS5j*",
+          "NOT /click/*/aHR0cHM6Ly9hcnRzeS5uZXQv*",
+          "NOT /click/*/aHR0cHM6Ly9pdHVuZXMuYXBwbGUuY29tL3VzL3BvZGNhc3QvYXJ0c3kv*",
           "NOT /collections",
+          "NOT /editorial/*",
+          "NOT /forgot",
+          "NOT /gender-equality*",
+          "NOT /identity-verification*",
           "NOT /log_in",
+          "NOT /news",
+          "NOT /news/*",
+          "NOT /price-databse",
+          "NOT /reset_password",
+          "NOT /series/*",
           "NOT /sign_up",
-          "NOT /click/*/aHR0cHM6Ly9pdHVuZXMuYXBwbGUuY29tL3VzL3BvZGNhc3QvYXJ0c3kv*",
-          "NOT /click/*/aHR0cHM6Ly9hcnRzeS5uZXQv*",
-          "NOT /click/*/aHR0cHM6Ly9hcnRzeS50eXBlZm9ybS5j*",
-          "NOT /click/*/aHR0cHM6Ly9hcHAuYWRqdXN0LmNv*",
           "NOT /users/auth/facebook/callback*",
-          "NOT /identity-verification*",
-          "NOT /reset_password",
-          "NOT /change-password",
-          "NOT /forgot",
-          "NOT *L2FydGljbGVz*",
-          "NOT *L2FydGljbGUv*",
```

Comment:
@MounirDhahri This PR removes these two entries completely. Is that intended?

Reply:
I also noticed that. These two has been deleted after I ran `yarn build`

Reply:
Ok